xfoil_module/routines.py

```diff
@@ -2,30 +2,22 @@
 import logging
 import numpy as np
 from xfoil import XFoil
-# from vgfoil import VGFoil
-
 
 logger = logging.getLogger(__name__)
 
-# TODO properly code initialization of VGFoil or XFoil instance and property setters
-xf = XFoil()  # create xfoil object
-xf.print = 1  # Suppress terminal output: 0, enable output: 1
-#xf.ctauvg = (0, 0, 0.25, 2.5)
-#xf.xvg = (1, 0.8)
-#xf.hvg = (0, 0.002)
-#xf.xtr = (0.05, 0.1)
-
 
 def call(analysis):
     '''Runs analysis using Xfoil'''
-    logger.info("Xfoil solver started.")
+    # TODO properly code initialization of XFoil Class instance and property setters
+    xf = XFoil()  # create xfoil object
+    xf.print = int(not analysis.configuration_values["silent_mode"])  # Suppress terminal output: 0, enable output: 1
     # load airfoil shapefiles dataset
     input_dataset = analysis.input_manifest.get_dataset("aerofoil_shape_data")
+    airfoil_file_path = input_dataset.files.filter(name=analysis.input_values["airfoil_geometry_filename"]).one().path
```

Comment:
It would be better to use `.local_path` here to avoid ambiguity. I'm probably going to deprecate use of `path` at some point (it's slightly difficult to remove currently)

Reply:
See https://github.com/octue/octue-sdk-python/issues/470 for more info